api/api/version_endpoints_api_test.go

```diff
@@ -3617,6 +3617,121 @@ func TestUpdateEndpoint(t *testing.T) {
 				data: Error{Message: "custom predictor image must be set"},
 			},
 		},
+		{
```

Comment:
Can you please add one more test where the deployment status is running and it change the deployment type?

Reply:
Done